packages/otel/src/instrumentation.ts

```diff
@@ -1,22 +1,23 @@
-import { type ClientRequest } from "node:http";
+import { SpanStatusCode } from "@opentelemetry/api";
+import { W3CTraceContextPropagator } from "@opentelemetry/core";
 import { HttpInstrumentation } from "@opentelemetry/instrumentation-http";
 import { Resource } from "@opentelemetry/resources";
 import { NodeSDK } from "@opentelemetry/sdk-node";
 import {
   SemanticAttributes,
   SemanticResourceAttributes,
 } from "@opentelemetry/semantic-conventions";
-// import { diag, DiagConsoleLogger, DiagLogLevel, trace } from "@opentelemetry/api";
-import { W3CTraceContextPropagator } from "@opentelemetry/core";
-import { SpanStatusCode } from "@opentelemetry/api";
-import { batchSpanProcessor } from "./otel-traces-setup";
+import { type ClientRequest } from "node:http";
 import { otelLogsProcessor } from "./otel-logs-setup";
+import { batchSpanProcessor } from "./otel-traces-setup";
```

Comment:
unused

Reply:
I just checked and there is no lint/tsc errors - it is used down below as `spanProcessor` (not part of the diff)